crm/include/crm.inc.php

```diff
@@ -24,7 +24,7 @@
 $crm_version = array(
     'major' => 0
     , 'minor' => 4
-    , 'patch' => 6
+    , 'patch' => 7
```

Comment:
Would this be a big enough change to warrant moving up the minor version number by one?

Reply:
probably, yes, though @elplatt had previously mentioned that because the project was in maintenance only mode rather than active development, & it was a DB change, that he wasn't happy pushing this change